Class extends value [object Module] is not a constructor or null

Hi,

After updating to 4.1.1 I am getting this error

```ts
TypeError {▼
  name: 'TypeError',
  message: 'Class extends value [object Module] is not a constructor or null',
  stack: 'Class extends value [object Module] is not a constructor or null\n' +
  'at <anonymous> (/app/node_modules/.pnpm/node-cron@4.1.1/node_modules/node-cron/dist/esm/tasks/inline-scheduled-task.js:14:36)\n' +
  '    at ModuleJob.run (node:internal/modules/esm/module_job:274:25)\n' +
  '    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:644:26)\n' +
  '    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)',
  unhandled: false,
  [Symbol(Symbol.toStringTag)]: 'Error',
}
```

This is how I run cron

```js
import cron from 'node-cron'
import sitemap from '~/utils/sitemap'

export default defineNitroPlugin(() => {
  const pmId = process.env.pm_id

  if (pmId === 1 || pmId === undefined) {
    const config = useRuntimeConfig()

    // run only on one pm2 instance or undefined for local
    // one time run
    sitemap(config)

    // everyday at midnight
    cron.schedule('0 0 * * *', () => {
      console.log('executing daily midnight cron')
      sitemap(config)
    })
  }
})

```

Is there actually any update on this one? I am facing the same issue.

Various AI tools suggested this as an alternative:
```
import { createRequire } from 'node:module';
const require = createRequire(import.meta.url);
const cron = require('node-cron');
```

which sort of works, but not really. I mean, I have that working in dev, but the minute I try to build the Nuxt app for prod, I start getting these `Cannot find module 'node-cron'` and I haven't been able to figure this one out yet. All possible suggestions I got and tried weren't really fruitful at all.

--- a/src/tasks/inline-scheduled-task.ts
+++ b/src/tasks/inline-scheduled-task.ts
@@ -1,4 +1,4 @@
-import EventEmitter from "events";
+import { EventEmitter } from "events";
 import { Execution, ScheduledTask, TaskContext, TaskEvent, TaskFn, TaskOptions } from "./scheduled-task";
 import { Runner, RunnerOptions } from "../scheduler/runner";
 import { TimeMatcher } from "../time/time-matcher";

